package.json

```diff
@@ -71,10 +71,10 @@
     "react-query": "3.39.1",
     "react-reflex": "^4.0.0",
     "react-router-dom": "^5.3.3",
-    "react-select": "^4.3.1",
+    "react-select": "^5.4.0",
```

Comment:
Upgraded to fix type issues. Breaking changes were minor--mostly renaming props and types